scripts/geoint-deps-repo.sh

```diff
@@ -40,11 +41,11 @@ BKvRcPtz0vU2ZA+JjgCXqpu0sLpMY/CGlgn9m4XlZYQ8ok/fe1no7k6cJat1EHLH
 EOF
 
 cat > "${GEOINT_DEPS_REPO}" <<EOF
-[geoint-deps-${GEOINT_DEPS_CHANNEL}]
+[geoint-deps-${GEOINT_DEPS_CHANNEL/\//-}]
 name = GEOINT Dependencies
 baseurl = ${GEOINT_DEPS_BASEURL}
 enable = 1
-gpgcheck = 1
+gpgcheck = ${GEOINT_DEPS_GPGCHECK}
 gpgkey=file://${GEOINT_DEPS_KEY}
-repo_gpgcheck = 1
+repo_gpgcheck = ${GEOINT_DEPS_GPGCHECK}
```

Comment:
Great addition, I always thought about doing this and never got around to it, hah!